Add UI WG meeting notes for 13032022 (#172)
moja-global/About_moja_global: working-groups/user-interface/MeetingNotes/13-5-22.md

## Attendees
- Akash Jha
- Anjali Kushwaha
- Arpita Naval
- Gunjan Chhablani
- Harshit Jain
- Mohammed Saifullah
- Padmaja Bhol
- Rajat Panwar
- Ritika Lohia
- Sanjay Singh Rajpoot
- Shloka Gupta
- Sukriti Sood
- Yash Kandalkar

## Agenda
- Finalize the content under the respective sections. Talk about the missing pieces of the puzzle.
- Brainstorm ways to conduct usability testing.
- Talk about ways to improve on a priority basis.

## Discussion
- TSC WG Notice Board
    - Putting it up on community website.
    - Sukriti created PR regarding WG structure. A demo is present here: https://github.com/moja-global/community-website/pull/304

- Flint-UI
    - Yash has made some components in Ant Design.
    - Figma design for GCBM not present.
    - Figma design shows flow for RothC and Point.
        - Need to get issues fixed.
    - The components are ready, but need to import in the file.
    - Project created on the repository for Point/RothC.
    - Need the components in order, first datepicker, then others.
    - Already created issues for some independent components and we have assignees.
        - Need to check with them whether they have a problem or not.

> - [GSoC - UI Projects](https://github.com/moja-global/mentorship/blob/main/google-summer-of-code/GSOC-2022-Ideas.md)

Reviewer: ```suggestion
- [Gcbm - UI Project](https://github.com/moja-global/mentorship/blob/main/google-summer-of-code/GSOC-2022-Ideas.md)
```

>     -  GCBM Simulation Editor for FLINT-UI
>         -  moja global is telling the countries how to use land in an efficient way.
>         -  FLINT is used for this purpose, so they can reduce their greenhouse gas emissions.

Reviewer: ```suggestion
```

        -  GCBM is built on top of flint. It is an important
        -  We want FLINT-UI for people from non-programming backgrounds to be able to use the FLINT.

>         - FLINT has PointC and RothC, and GCBM.

Reviewer: ```suggestion

```

>         - PointC and RothC are simpler methods, GCBM is complex.

Reviewer: ```suggestion
        - PointC and RothC are simpler and will help a person get a hold on the codebase, GCBM is complex. It's like diving into a shallow pool before a deep pool.
```

>         - GCBM has 10 building blocks, and a person should be able to experiment with those building blocks.

Reviewer: ```suggestion
        - GCBM's UI (For example) should have 10 building blocks, and a person should be able to experiment with those building blocks.
```

>             - Currently we are only giving the user three steps and we constrain it.

Reviewer: ```suggestion
            - Currently we are only giving the user three blocks and we constrain the user not allowing them to experiment
```

>             - We want to provide a way to configure the 10 building blocks.

Reviewer: ```suggestion
            - By the end of the project we want to provide a way to experiment with all the 10 building blocks.
```

>         - Visualization of the model - input files.

Reviewer: ```suggestion
        - Visualization of the model - input files. Download the GCBM demo run
```

- Why Ant-Design instead of tail-wind? See this [discussion](https://github.com/moja-global/FLINT-UI/discussions/173).
    - Reproducible Components
        - Ant-Design allows us to use same components throughout the UI.
        - We will use pre-build designs eventually with some modifications.
    - Customization
        - More customization is possible in Ant-Design.

## Check in on previous action items
- [x] Sukriti to create a PR for the working groups section on community website.
- [ ] Have the active working groups on the website - PR needs to be merged.
    - [x] Sukriti to add issues for each working group.
    - [x] Focus on only the active working groups:
        - [x] Documentation
        - [x] UI
            - [ ] ~Arpita to add the UI working group (About) to community website.~
- [ ] ~Amit to write content for the Mentorship working group.~
- [x] Yash and Sanjay to work on the components for FLINT-UI.

## New Action items
- [ ] Sukriti to merge TSC WG PR.
- [ ] Hold a closing note meeting for TSC WG noticeboard.
    - [ ] Discuss the project, what all can be improved, etc.

> - [ ] Start with RothC and Point from the next meeting.

Reviewer: ```suggestion
- [ ] Start with the next project next week.
```
